apps/admin/frontend/src/components/full_test_deck_tally_report_button.test.tsx

```diff
@@ -58,3 +69,16 @@ test('prints appropriate report for general election', async () => {
     expect(printOptions).toMatchObject({ sides: 'one-sided' });
   });
 });
+
+test('renders SaveFileToUsb component for saving PDF', async () => {
+  const usbDrive = mockUsbDrive('mounted');
+  const { unmount } = renderInAppContext(<FullTestDeckTallyReportButton />, {
+    electionDefinition: electionFamousNames2021Fixtures.electionDefinition,
+    usbDrive,
+  });
+  userEvent.click(screen.getByText('Save Full Test Deck Tally Report as PDF'));
+  const modal = await screen.findByRole('alertdialog');
+  within(modal).getByText('Save Test Deck Tally Report');
+
+  unmount();
```

Comment:
Ooh interesting, were you seeing issues without this explicit `unmount`? I _think_ unmounting happens automatically in between tests: https://testing-library.com/docs/react-testing-library/api/#cleanup

Reply:
This test was failing and causing the rest of the suite to fail. I added the `unmount` to try to fix that, but I'm not actually sure if it did since I also fixed the test failure itself. Deleted `unmount` in https://github.com/votingworks/vxsuite/pull/3072/commits/3c83f9d230db1d9f869a4f9ce90f6af2ee520a37#diff-2c47912ea89be851061ffeabf15fa855fbaa8b9e8282e4128cdcfeb5452607b4L83